docs/configuration.md

```diff
@@ -190,8 +190,10 @@ of the most common options to set are:
     and it is up to the application to avoid exceeding the overhead memory space
     shared with other non-JVM processes. When PySpark is run in YARN or Kubernetes, this memory
     is added to executor resource requests.
-
-    NOTE: Python memory usage may not be limited on platforms that do not support resource limiting, such as Windows.
+    <br/>
+    <em>Note:</em> Python memory usage may not be limited on platforms that do not support resource limiting, such as Windows.
+    <br/>
+    <em>Note:</em> Python memory usage is dependent on Python's 'resource' module; therefore, the behaviors and limitations are inherited.
```

Comment:
Why is this a separate note? It says the same thing as the one above, but adds that the `resource` module is used. I think what you want is to make the note above more specific, by changing it to this:

```
<em>Note:</em> Python memory usage may not be limited on platforms that do not support resource limiting (specifically, Python's `resource` module), such as Windows.
```

Reply:
Yup

Reply:
Eh, actually I intended to explicitly this feature is dependent on `resource` module so that we don;t happen to fix the doc anymore by this problem. Let me try to combine it anyway.

Reply:
One difference between Windows and the current case is that, it says the memory limit is set:

```
Current mem limits: 9223372036854775807 of max 9223372036854775807

Setting mem limits to 1048576 of max 1048576
```

but it's not set.

Reply:
Then it should state that it is attempting to set memory limits?